webapp/store/snap_details_views.py

```diff
@@ -1,7 +1,8 @@
 import flask
 
-import bleach
 import humanize
+import bleach
+from ruamel.yaml import YAML
```

Comment:
Import not needed anymore.

Reply:
This is still relevant, no? This file hasn't been changed?